test/xpu_api/numerics/complex.number/complex.special/double_long_double_explicit.pass.cpp

```diff
@@ -0,0 +1,33 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <complex>
+
+// template<> class complex<double>
+// {
+// public:
+//     explicit constexpr complex(const complex<long double>&);
+// };
+
+#include "support/test_complex.h"
+
+ONEDPL_TEST_NUM_MAIN
+{
+    IF_LONG_DOUBLE_SUPPORT_IN_COMPILETIME(const dpl::complex<long double> cd(2.5, 3.5);
+                                          dpl::complex<double> cf(cd);
+                                          assert(cf.real() == cd.real());
+                                          assert(cf.imag() == cd.imag()))
+#if TEST_STD_VER >= 11
```

Comment:
I think we don't support such old C++ (< 11)  versions....

Reply:
All these check is as is from llvm tests.
Do you think that really required to have additional changes in these places?